OracleGoldenGate/23/nginx/nginx.conf

```diff
@@ -0,0 +1,90 @@
+## Copyright (c) 2021, Oracle and/or its affiliates.
+
+##
+##  nginx.conf
+##  Configuration file for Nginx Reverse Proxy
```

Comment:
The correct way to format NGINX is in all capitals as per their trademark documentation (see other comment for the URL). While I'm being pedantic, "reverse proxy" shouldn't be capitalised here either. :)